.github/workflows/roundtrip/encrypt-decrypt.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+cd ../..
+pwd
+
+mv ../../../opentdf-cli-*.tgz ../../../cli
+npm uninstall @opentdf/cli && npm ci && npm i ../../../cli/opentdf-cli-*.tgz
+
+touch ./sample.txt
+echo "Hello World!" >> ./sample.txt
+
+npx @opentdf/cli --log-level DEBUG \
+--kasEndpoint http://localhost:65432/api/kas \
+--oidcEndpoint http://localhost:65432 \
+--auth tdf:tdf-client:123-456 \
+--output sample.txt.tdf encrypt sample.txt \
+--attributes https://example.com/attr/Classification/value/S,https://example.com/attr/COI/value/PRX
+
+npx @opentdf/cli --log-level DEBUG \
+--kasEndpoint http://localhost:65432/api/kas \
+--oidcEndpoint http://localhost:65432 \
+--auth tdf:tdf-client:123-456 \
+--output sample_out.txt decrypt sample.txt.tdf
+
+diff sample.txt sample_out.txt
```

Comment:
It's a very good sign and a testament to the CI work you guys have been doing that adding this e2e test is basiclaly + 25 lines.